Source/control.cpp

```diff
@@ -213,33 +213,33 @@ char SpellITbl[] = {
 };
 
 enum panel_button_id {
-	PANBTN_CHARINFO,
-	PANBTN_QLOG,
-	PANBTN_AUTOMAP,
-	PANBTN_MAINMENU,
-	PANBTN_INVENTORY,
-	PANBTN_SPELLBOOK,
-	PANBTN_SENDMSG,
-	PANBTN_FRIENDLY,
+	kPanbtnCharinfo,
+	kPanbtnQlog,
+	kPanbtnAutomap,
+	kPanbtnMainmenu,
+	kPanbtnInventory,
+	kPanbtnSpellbook,
+	kPanbtnSendmsg,
+	kPanbtnFriendly,
 };
 
-/** Maps from panel_button_id to the position and dimensions of a panel button. */
-int PanBtnPos[8][5] = {
+/** Positions of panel buttons. */
+SDL_Rect PanBtnPos[8] = {
 	// clang-format off
-	{   9,   9, 71, 19, true  }, // char button
-	{   9,  35, 71, 19, false }, // quests button
-	{   9,  75, 71, 19, true  }, // map button
-	{   9, 101, 71, 19, false }, // menu button
-	{ 560,   9, 71, 19, true  }, // inv button
-	{ 560,  35, 71, 19, false }, // spells button
-	{  87,  91, 33, 32, true  }, // chat button
-	{ 527,  91, 33, 32, true  }, // friendly fire button
+	{   9,   9, 71, 19 }, // char button
+	{   9,  35, 71, 19 }, // quests button
+	{   9,  75, 71, 19 }, // map button
+	{   9, 101, 71, 19 }, // menu button
+	{ 560,   9, 71, 19 }, // inv button
+	{ 560,  35, 71, 19 }, // spells button
+	{  87,  91, 33, 32 }, // chat button
+	{ 527,  91, 33, 32 }, // friendly fire button

```

Comment:
Why the change here? What was the last parameter before?

Reply:
It's unused, it made clang-tidy grumpy and this was better handled as an `SDL_rect` anyway so that was the easy way to deal with the issue.